README.md

```diff
@@ -49,8 +49,13 @@ If you're completly new to the world of P4 and are looking for step-by-step inst
 ### Tools
 
 - [gNMI CLI Tool](/stratum/tools/gnmi/README.md)
+- [Bf Pipeline Builder](/stratum/hal/bin/barefoot/README.pipeline.md#stratum-bfpipelineconfig-format-and-the-bfpipelinebuilder)
```

Comment:
Maybe use `BF` or `Barefoot` ?

Reply:
Good point, I'll go with Tofino since Barefoot is Intel now/soon.